docs/release-train-issue-template.md

```diff
@@ -0,0 +1,76 @@
+Release Akka HTTP $VERSION$
+
+<!--
+Release Train Issue Template for Akka HTTP
+
+(Liberally copied and adopted from Scala itself https://github.com/scala/scala-dev/blob/b11cd2e4a4431de7867db6b39362bea8fa6650e7/notes/releases/template.md)
+
+For every release, use the `scripts/create-release-issue.sh` to make a copy of this file named after the release, and expand the variables.
+
+Variables to be expanded in this template:
+- VERSION=???
+
+Key links:
+  - akka/akka-http milestone: https://github.com/akka/akka-http/milestone/?
+-->
+
+### Cutting the release
+
+- [ ] Check that open PRs and issues assigned to the milestone are reasonable
+- [ ] For minor or major versions, update the Change date in the LICENSE file and update the `licenses` url in the build.
+- [ ] For minor or major versions, add a release notes entry in `docs/src/main/paradox/release-notes/`.
```

Comment:
@johanandren I adjusted this according to https://github.com/akka/akka-http/pull/4183